docs/languages.rst

```diff
@@ -16,25 +16,50 @@ See :ref:`template-language-guide` until this section is more complete.
 Using a Different Variable-Length Array Type
 ==============================================
 
-For now this tip is important for people using the experimental C++ support. To use :code:`std::vector` instead of the
-minimal build-in :code:`variable_length_array` type create a properties override yaml file and pass it to nnvg.
+For now this tip is important for people using the experimental C++ support.  To set which variable length array
+implementation to use, create a properties override yaml file and pass it to nnvg.  Specifying the use of an
+allocator is optional (If ctor_convention is set to "default" then the allocator_include and allocator_type
+properties don't need to be set.)
 
-vector.yaml
+Alternatively, you may specify the language standard argument as -std=c++17-pmr or -std=cetl++ as short-hand for
+the following configurations shown below.
+
+c++17-pmr.yaml
+"""""""""""""""""
+
+.. code-block :: yaml
+
+    nunavut.lang.cpp:
+      options:
+        variable_array_type_include: "<vector>"
+        variable_array_type_template: "std::vector<{TYPE}, {REBIND_ALLOCATOR}>"
+        variable_array_type_constructor_args: ""
+        allocator_include: "<memory>"
+        allocator_type: "std::pmr::polymorphic_allocator"
+        allocator_is_default_constructible: true
+        ctor_convention: "uses-trailing-allocator"
+
+cetl++.yaml
```

Comment:
Please add to this document that `cetl++ == c++14` but with extra stuff from CETL.